src/google/protobuf/compiler/objectivec/objectivec_helpers.h

```diff
@@ -315,7 +316,6 @@ class PROTOC_EXPORT ImportWriter {
                                   const std::string& runtime_import_prefix,
                                   bool default_cpp_symbol = false);
 
- private:
```

Comment:
I wanted to reuse this collector since the format of the `objc_class_prefix_mappings` file is the same as the one for framework mappings.